Aquality.Selenium/src/Aquality.Selenium/Browsers/BrowserTabNavigation.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Aquality.Selenium.Core.Localization;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Aquality.Selenium.Browsers
+{
+    public class BrowserTabNavigation : IBrowserTabNavigation
+    {
+        private readonly RemoteWebDriver driver;
+
+        internal BrowserTabNavigation(RemoteWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private ILocalizedLogger Logger => AqualityServices.LocalizedLogger;
+
+        public void CloseTab()
+        {
+            Logger.Info("loc.browser.tab.close");
+            driver.Close();
+        }
+
+        public string GetTabName()
+        {
+            Logger.Info("loc.browser.get.tab.name");
+            return driver.CurrentWindowHandle;
+        }
+
+        public IList<string> GetTabNames()
+        {
+            Logger.Info("loc.browser.get.tab.names");
+            return driver.WindowHandles;
+        }
+
+        public void OpenNewTab(bool switchToNew = false)
+        {
+            Logger.Info("loc.browser.tab.open.new");
+            AqualityServices.Browser.ExecuteScript(JavaScript.OpenNewTab);
+            if (switchToNew)
+            {
+                SwitchToNewTab();
+            }
+        }
+
+        public void SwitchToNewTab(bool closeCurrent = false)
```

Comment:
should all of these methods be "chaining"?

Reply:
not sure understand what you mean, could you provide an example

Reply:
@mialeska  why?

Reply:
@knysh https://www.codeproject.com/articles/640997/fluent-interfaces-and-method-chaining-in-csharp
@panihimovsky to do things like OpenNewTab().SwitchToNewTab() in chain instead of parametrization. Nevermind, just asking